content/id/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,72 @@
+---
+reviewers:
+- davidopp
+- lavalamp
+title: Ikhtisar Administrasi Kluster
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+Ikhtisar administrasi kluster ini ditujukan untuk siapapun yang akan membuat atau mengelola kluster Kubernetes.
+Diharapkan untuk memahami beberapa [konsep](/docs/concepts/) dasar Kubernetes sebelumnya.
+{{% /capture %}}
+
+{{% capture body %}}
+## Perencanaan Kluster
+
+Lihat panduan di [Memilih Solusi yang Tepat](/docs/setup/pick-right-solution/) untuk mempelajari beberapa contoh tentang bagaimana merencanakan, mengatur dan mengonfigurasi kluster Kubernetes. Solusi yang akan dipaparkan di bawah ini disebut *distro*.
+
+Sebelum memilih panduan, berikut adalah beberapa hal yang perlu dipertimbangkan:
+
+ - Apakah kamu hanya ingin mencoba Kubernetes pada komputermu, atau kamu ingin membuat sebuah kluster dengan *high-availability*, *multi-node*? Pilihlah distro yang paling sesuai dengan kebutuhanmu.
+ - **Jika kamu merencanakan kluster dengan *high-availability***, pelajari bagaimana cara mengonfigurasi [kluster pada *multiple zone*](/docs/concepts/cluster-administration/federation/).
+ - Apakah kamu akan menggunakan **Kubernetes kluster di *hosting***, seperti [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), atau ***hosting* sendiri klustermu**?
+ - Apakah klustermu berada pada ***on-premises***, atau **di cloud (IaaS)**? Kubernetes belum mendukung secara langsung kluster hibrid. Sebagai gantinya, kamu dapat membuat beberapa kluster.
+ - **Jika kamu ingin mengonfigurasi Kubernetes *on-premises***, pertimbangkan [model jaringan](/docs/concepts/cluster-administration/networking/) yang paling sesuai.
+ - Apakah kamu ingin menjalankan Kubernetes pada **"bare metal" *hardware*** atau pada ***virtual machines* (VMs)**?
+ - Apakah kamu **hanya ingin mencoba kluster Kubernetes**, atau kamu ingin ikut aktif melakukan **pengembangan kode dari proyek Kubernetes**? Jika jawabannya yang terakhir, pilihlah distro yang aktif dikembangkan. Beberapa distro hanya menggunakan rilis *binary*, namun menawarkan lebih banyak variasi pilihan.
+ - Pastikan kamu paham dan terbiasa dengan beberapa [komponen](/docs/admin/cluster-components/) yang dibutuhkan untuk menjalankan sebuah kluster.
+
+Catatan: Tidak semua distro aktif dikelola. Pilihlah distro yang telah diuji dengan versi terkini dari Kubernetes.
+
+## Mengelola Kluster
+
+* [Mengelola kluster](/docs/tasks/administer-cluster/cluster-management/) akan menjabarkan beberapa topik terkait *lifecycle* dari kluster: membuat kluster baru, melakukan *upgrade* pada *node master* dan *worker*, melakukan pemeliharaan *node* (contoh: *upgrade* kernel), dan melakukan *upgrade* versi Kubernetes API pada kluster yang sedang berjalan.
+
+* Pelajari bagaimana cara [mengatur *node*](/docs/concepts/nodes/node/).
+
+* Pelajari bagaimana cara membuat dan mengatur [kuota *resource*](/docs/concepts/policy/resource-quotas/) untuk *shared* kluster.
```

Comment:
maybe add kuota resource (*resource quota*) since `resource quota` is Kubernetes field/feature